homeassistant/components/blink/strings.json

```diff
@@ -117,5 +117,18 @@
     "not_loaded": {
       "message": "{target} is not loaded"
     }
+  },
+  "issues": {
+    "service_depreciation": {
```

Comment:
depreciation is a different word than deprecation and has a different meaning here.

Reply:
Oh boy - guilty of just copying (and I just fixed the previous comment). 
https://github.com/home-assistant/core/blob/a823edf1c2743a7f91f97ab30d327facac00fba4/homeassistant/components/ezviz/camera.py#L256-L265